packages/cli/commands/project/migrateApp.js

```diff
@@ -168,12 +168,16 @@ exports.handler = async options => {
       );
       process.exit(EXIT_CODES.SUCCESS);
     }
-  } catch (e) {
+  } catch (error) {
     SpinniesManager.fail('migrateApp', {
       text: i18n(`${i18nKey}.migrationStatus.failure`),
       failColor: 'white',
     });
-    logApiErrorInstance(e.error, new ApiErrorContext({ accountId }));
+    if (error.error) {
+      logApiErrorInstance(error.error, new ApiErrorContext({ accountId }));
```

Comment:
Not a blocker, but you could shorten this to `error.error || error` in logApiErrorInstance() if you want to cut down on lines of code

Reply:
Nice! Addressed in https://github.com/HubSpot/hubspot-cli/pull/1089/commits/210549adfc36170bd25df9d6dcb2abde6036b3b8